cmd/meroxa/root/fixtures/sync.go

```diff
@@ -0,0 +1,160 @@
+package fixtures
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/meroxa/cli/cmd/meroxa/builder"
+	"github.com/meroxa/cli/cmd/meroxa/turbine"
+	"github.com/meroxa/cli/log"
+	"github.com/meroxa/meroxa-go/pkg/meroxa"
+)
+
+var (
+	_ builder.CommandWithDocs    = (*Sync)(nil)
+	_ builder.CommandWithArgs    = (*Sync)(nil)
+	_ builder.CommandWithFlags   = (*Sync)(nil)
+	_ builder.CommandWithClient  = (*Sync)(nil)
+	_ builder.CommandWithLogger  = (*Sync)(nil)
+	_ builder.CommandWithExecute = (*Sync)(nil)
+)
+
+const FixtureFilePermissions = 0644
+
+type introspectResourceClient interface {
+	IntrospectResource(ctx context.Context, nameOrUUID string) (*meroxa.ResourceIntrospection, error)
+}
+
+type Sync struct {
+	client introspectResourceClient
+	logger log.Logger
+
+	args struct {
+		NameOrUUID string
+	}
+
+	flags struct {
+		Collections string `long:"collections" short:"c" usage:"--collections table1,table2"`
+		Path        string `long:"path" short:"p" usage:"--path ./foo/bar"`
+	}
+}
+
+func (f *Sync) Usage() string {
+	return "sync [NAMEorUUID]"
```

Comment:
Syncing fixtures doesn't require the app name and instead actually relies on the resource name (e.g. the resource to sync fixtures for) and optionally the collections to sync.

The "app" isn't really even used (or even a dependency). It currently locates the `/fixtures` dir from within the current path (or the passed in path).

Reply:
@ahmeroxa oh! in that case then maybe we should make that via the `--resource` flag instead if we make `fixtures` one of the commands or maybe make the command live under `resources`. That way, the argument being a resource name would be super clear in my opinion.

I'm also thinking that maybe `sync` is not the right term considering we would only _pull_ rather than push as well. (i.e.: it's not bi-directional). I wonder if just `fixtures` could do it:

```
$ meroxa resources fixtures RESOURCE_NAME
```

What do you think?

Reply:
Or maybe `$ meroxa resources export-fixtures`... not in love, but much clearer.

Reply:
We could alternatively replace the term `fixtures` entirely. Maybe `samples` is clearer. Then you would `sample` a resource.

Reply:
@ahmeroxa i like that idea. To ship this, let me know how you'd like to proceed. I'm happy to pair on this and get it out of the door if you want.